tests/unit_tests/blas/include/reference_blas_templates.hpp

```diff
@@ -22,6 +22,7 @@
 
 #include <stdlib.h>
 #include <complex>
+#include <exception>
```

Comment:
I noticed that for other exceptions, we include and throw exceptions that are defined here: [exceptions_helper.hpp](https://github.com/oneapi-src/oneMKL/blob/develop/src/include/exceptions_helper.hpp) 

Should I follow this for this exception as well?

Reply:
I think adding a new exception in that file may be a good idea. @mkrainiuk is this the recommended way?